src/main/java/nl/cherement/jak/service/UserService.java

```diff
@@ -15,6 +18,16 @@ public UserService(UserRespository repository) {
         super(repository);
     }
 
+    public Hashtable<Long, String> findAllShortened() {
+        Hashtable<Long, String> users = new Hashtable<>();
+        List<UserEntity> allUsers = repository.findAll();
+
+        for (UserEntity user : allUsers) {
+            users.put(user.getId(), user.getUsername());
+        }
+        return users;
+    }
```

Comment:
Dit kan afgehandeld worden in de repository met `findAllByOrderById`

Reply:
Maar dan krijg je alles ipv alleen de id en username toch?